CHANGELOG.md

```diff
@@ -1,9 +1,25 @@
 # Unreleased
 
+# 9.13.0 (Feature release)
+
+## New features
+
+### Opt in to the new GOV.UK Frontend link styles
+
+Links now have underlines that are consistently thinner and a bit further away from the link text.
+
+Links also have a clearer hover state, where the underline gets thicker to make the link stand out to users.
+
+The new link styles are opt-out by default because [Chromium browsers currently cannot display them](https://github.com/alphagov/govuk-frontend/issues/2204).
```

Comment:
Great idea linking to the known issue, but I don't think this summary is quite right – Chromium browsers can display them, it's just there's an issue with links that are inside a multi-column layout.

I'm also tripping up over 'opt-out by default' – I'd personally would have said they're opt-in by default! I think there's ambiguity as to whether this means 'by default **you are** opted out' or 'by default **you have to** opt-out'?

Reply:
I think this comment still needs addressing.

Reply:
Ah, sorry - I tried to address it in [this commit](https://github.com/alphagov/govuk-prototype-kit/pull/1013/commits/88b79f46a2c2b5c173e1f12ab807762d79eb4044) a few hours ago.